episodes/anonymization.md

```diff
@@ -32,19 +33,352 @@ Metadata elements in imaging, such as patient names and addresses, are often cle
 
 DICOM files contain metadata, which includes various types of identifying information that should remain confidential. The easiest way to mitigate issues with DICOM metadata is to avoid having it in the first place. If possible, opt to receive just the images and select metadata rather than the entire DICOM file. When sharing data with collaborators, there is often no need to share the full DICOM files.
 
+
+### Text on Images
+
+Occasionally, technicians will burn information directly onto images as part of a burned-in annotation. This may include details such as diagnoses, demographics, or the patient's name. Fortunately, this text is usually typed rather than handwritten, making it recognizable by optical character recognition (OCR) functions. Often, this text is placed away from the center of the image, allowing for clever cropping to eliminate it entirely in some datasets.
+
+
+
+::::::::::::::::::::::::::::::::::::::: challenge
+
+## Challenge: Getting rid of identifying burned in data
+
+An ultrasound (scraped from the public internet on a creative commons license lisence [here](https://www.flickr.com/photos/jcarter/2461223727).) must be kept at it's existing height- width dimensions. You should resize away from RGB to grayscale. It can be opened as follows:
+
+```python
+
+import numpy as np
+import matplotlib.pyplot as plt
+from skimage import io, filters
+
+image = 'data/anonym/identifiable_us.jpg'
+io.imshow(image)
+io.show()
+```
+
+```output
+```
+
+![Image from flikr website published with a permissive lisence.](fig/identifiable_us.jpg){alt='Identifiable ultrasound'}
+
+Write code for two approaches that de-identify the ultrasound. 
+
+
+::::::::::::::: solution
+
+## Solution
+
+The image  has identifying metdata. You should identify not only the name, but the date and place as problematic.  You can take three different approaches. One would be to mask the data, the other would be to blur it. First we will show a blurred image:
```

Comment:
What is the third approach?

Reply:
It's under "Finally, you could always just chop off the offending part so to speak, and resize:"

Reply:
Yes, but I find a bit odd that the previous sentence says "You can take three different approaches" and immediately after only two are listed.

Reply:
Fixed now